src/mango/test/mango.py

```diff
@@ -44,7 +44,7 @@ def url(self):
         return "http://{}:{}/{}".format(self.host, self.port, self.dbname)
 
     def path(self, parts):
-        if isinstance(parts, (str, unicode)):
+        if isinstance(parts, ("".__class__, u"".__class__)):
```

Comment:
only part worth of note. Another way to do this is : 
```
try:
    basestring  # attempt to evaluate basestring
    def isstr(s):
        return isinstance(s, basestring)
except NameError:
    def isstr(s):
        return isinstance(s, str)
```

Reply:
Cute shortcut, makes perfect sense.